app/models/concerns/organization_scope.rb

```diff
@@ -1,9 +1,16 @@
 module OrganizationScope
   extend ActiveSupport::Concern
+  include ReportsKit::Model
 
   included do
     belongs_to :organization
 
     scope :for_organization, ->(organization) { where(organization: organization) }
+
+    reports_kit do
+      contextual_filter :for_organization, ->(relation, context_params){
+        relation.for_organization(context_params[:organization_id])
+      }
+    end
```

Comment:
Since this ReportsKit filter would likely apply to anything with a `for_organization` filter on it, this place seems convenient for it. I'm assuming right now that if we had other config options within a model that a 2nd config block would be appended vs. override this one. Something easy to check on before this PR is complete.